src/index.ts

```diff
@@ -206,8 +212,10 @@ function teenyRequest(
     options.body = createMultipartStream(boundary, multipart);
 
     // Multipart upload
+    teenyStatistics.requestStarting();
     fetch(uri, options).then(
       res => {
+        teenyStatistics.requestFinished();
```

Comment:
Should we fire this in the fetch promise error handlers as well?

Reply:
yes, yes we should. fixed.